torch/_inductor/choices.py

```diff
@@ -323,3 +327,69 @@ def score_fusion(
             memory_score,
             proximity_score,
         )
+
+    @staticmethod
+    def estimate_runtime(
+        counted_flops: int,
+        counted_bytes: int,
+        gpu_flops: int,
+        gpu_memory_bandwidth: int,
```

Comment:
Should we give a pointer to SIMDKernelFeatures, so people can come up with their own estimate?  The args here don't really give much flexibility, how would an autotuner get the features it neesd to make an interesting choice?